orderbook/src/event_updater.rs

```diff
@@ -21,15 +21,20 @@ const INSERT_EVENT_BATCH_SIZE: usize = 250;
 pub struct EventUpdater {
     contract: GPv2Settlement,
     db: Database,
+    last_handled_block: Option<u64>,
```

Comment:
nit: could be an [AtomicU64](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU64.html) to avoid the mutex on the whole EventUpdater.

Reply:
It could be, on the other hand I don't think it ever makes sense to call update events in parallel.